### PR TITLE
minor sp: exponentian --> exponential

### PR DESCRIPTION
**Pull request by brettkoonce**

### CODE REVIEW

File: Sources/TensorFlow/Optimizers/MomentumBased.swift [hunk 1]
```diff
@@ -208,8 +208,8 @@ where
 ///
 /// - Parameters:
 ///   - learningRate: A Float. The learning rate (default value: 1e-3).
-///   - beta1: A Float. The exponentian decay rate for the 1st moment estimates (default value: 0.9).
-///   - beta2: A Float. The exponentian decay rate for the 2nd moment estimates (default value: 
+///   - beta1: A Float. The exponential decay rate for the 1st moment estimates (default value: 0.9).
```
Comment (dan-zheng) [inline Sources/TensorFlow/Optimizers/MomentumBased.swift]: Thanks for the typo fix!

Actually, these doc comments have further room for improvement:
- No doc comments should repeat type information: remove `A Float`.
- The standard doc comments for default argument values is: ``The default value is `xxx`.``

```suggestion
///   - beta1: The exponential decay rate for the 1st moment estimates. The default value is `0.9`.
```

Comment (brettkoonce) [reply Sources/TensorFlow/Optimizers/MomentumBased.swift]: @dan-zheng let me know what you think!

File: Sources/TensorFlow/Optimizers/MomentumBased.swift [hunk 2]
```diff
@@ -207,13 +207,12 @@ where
...
 ///
 /// - Parameters:
-///   - learningRate: A Float. The learning rate (default value: 1e-3).
-///   - beta1: A Float. The exponential decay rate for the 1st moment estimates (default value: 0.9).
-///   - beta2: A Float. The exponential decay rate for the 2nd moment estimates (default value:
-///     0.999).
-///   - epsilon: A Float. A small scalar added to the denominator to improve numerical stability 
-///     (default value: 1e-8).
-///   - decay: A Float. The learning rate decay (default value: 0).
+///   - learningRate: Learning rate. The default value is `1e-3`.
```
Comment (dan-zheng) [inline Sources/TensorFlow/Optimizers/MomentumBased.swift]: A leading article ("the") is standard.
```suggestion
///   - learningRate: The learning rate. The default value is `1e-3`.
```

File: Sources/TensorFlow/Optimizers/MomentumBased.swift [hunk 3]
```diff
@@ -207,13 +207,12 @@ where
...
+///   - beta1: The exponential decay rate for the 1st moment estimates. The default value is `0.9`.
+///   - beta2: The exponential decay rate for the 2nd moment estimates. The default value is `0.999`.
+///   - epsilon: A small scalar added to the denominator to improve numerical stability.
+///     The default value is `1e-8`.
+///   - decay: Learning rate decay. The default value is `0`.
```
Comment (dan-zheng) [inline Sources/TensorFlow/Optimizers/MomentumBased.swift]:
```suggestion
///   - decay: The learning rate decay. The default value is `0`.
```

File: Sources/TensorFlow/Optimizers/MomentumBased.swift (hunk 2 above)
Comment (brettkoonce) [reply Sources/TensorFlow/Optimizers/MomentumBased.swift]: tweaked!

File: Sources/TensorFlow/Optimizers/MomentumBased.swift (hunk 3 above)
Comment (brettkoonce) [reply Sources/TensorFlow/Optimizers/MomentumBased.swift]: tweaked!

File: Sources/TensorFlow/Optimizers/MomentumBased.swift [hunk 4]
```diff
@@ -207,13 +207,12 @@ where
 /// (Kingma and Ba, 2014).
 ///
 /// - Parameters:
```
Comment (dan-zheng) [inline Sources/TensorFlow/Optimizers/MomentumBased.swift]: Sorry I overlooked this: this entire doc comment belongs on the `Adam.init` declaration, not on the `Adam` type declaration itself. Would you mind moving it there?

Comment (brettkoonce) [reply Sources/TensorFlow/Optimizers/MomentumBased.swift]: @dan-zheng like so?

Comment (dan-zheng) [reply Sources/TensorFlow/Optimizers/MomentumBased.swift]: Not quite - doc comments always belong above declarations, not in their bodies.
```
  /// - Parameters:
  /// ...
  public init(
    for model: __shared Model,
    learningRate: Float = 1e-3,
    beta1: Float = 0.9,
    beta2: Float = 0.999,
    epsilon: Float = 1e-8,
    decay: Float = 0
  ) {
```

Be sure to use `/// - Parameters:` with a leading hyphen. https://swift.org/documentation/api-design-guidelines/#write-doc-comment

Comment (brettkoonce) [reply Sources/TensorFlow/Optimizers/MomentumBased.swift]: @dan-zheng moved again, re-added the leading hyphen!

**Comment by saeta** [comment]
👍

**Pull request merged**